### DISCUSSION TITLE
How do I access my Pro subscription purchased on iOS from Android tablet?

### DISCUSSION THREAD

**Discussion by rachellawson** [osmandapp/OsmAnd#20004] [Q&A] [answered]
I have a new iPad tablet for navigation on the motorbike, a Carpe Iter, and want to use my OsmAnd Pro subscription, as detailed in https://osmand.net/docs/user/purchases/ios. 

<img src="https://github.com/osmandapp/OsmAnd/assets/807248/c03318aa-3134-4e33-9e75-469cbe296096"  width="300" height="auto"/> 

How do I do it??

**Comment by dmpr0** [comment]
Hello!
OsmAnd Pro's cross-platform capability is based on OsmAnd Cloud. First, you need to create an OsmAnd Cloud account on the device on which you purchased OsmAnd Pro, and then simply log in to the same OsmAnd Cloud account on other devices.

**Reply by rachellawson** [reply]
Hi

That's exactly what I did. Doesn't seem to be working

**Reply by rachellawson** [reply]
I know it's the same account as I've successfully pulled across my settings, custom maps etc. 
Can't get it to recognise the Pro account, though, so can't use those functions

**Reply by dmpr0** [reply]
Please try to open Menu – Settings – Purchases, tap on Restore purchases on needed device.

**Reply by rachellawson** [reply]
Yes, that's what I've been doing. 
A small notification appears saying "Purchases restored" but it is still set as Free account

**Reply by dmpr0** [reply]
Could you please contact me at dmitriy.prodchenko@osmand.net, provide the email you have used for OsmAnd Cloud account, I will check if it has an active OsmAnd Pro subscription attached?

**Comment by rachellawson** [comment, marked as answer]
We resolved this with a gift subscription on Android
